### ISSUE TITLE
Can't get Geoviews to show any output on Ubuntu 18.04

### ISSUE THREAD

**Issue by jacobhutchinson** [holoviz/geoviews#468]
No output is shown on Ubuntu 18.04, with or without GUI. Matplotlib itself works fine, but no output from Geoviews works at all. I am not sure if this is an environment issue, version issue or bug. I can include any code you want, but nothing outputs at all so I thought it may be redundant. Tested on a CLI only Ubuntu 18.04 Windows Subsystem for Linux 1 console and a full Ubuntu 18.04 VMware VM with GUI. I know Geoviews works to some extent, as it outputs file to .html fine. I am using Python 3.8.5, conda 4.8.3 and the following conda environment:
```
# packages in environment at /home/jacobhutchinson/miniconda3/envs/scraper:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
argparse                  1.4.0                    pypi_0    pypi
beautifulsoup4            4.9.1                    pypi_0    pypi
blas                      1.0                    openblas  
bokeh                     2.1.1                    pypi_0    pypi
boost-cpp                 1.72.0               h7b93d67_2    conda-forge
brotlipy                  0.7.0           py38h7b6447c_1000  
bzip2                     1.0.8                h7b6447c_0  
c-ares                    1.15.0            h7b6447c_1001  
ca-certificates           2020.6.24                     0  
cairo                     1.16.0            h3fc0475_1005    conda-forge
cartopy                   0.18.0           py38h4f3c6b6_2    conda-forge
certifi                   2020.6.20                py38_0  
cffi                      1.14.1           py38he30daa8_0  
cfitsio                   3.470                hf0d0db6_5  
chardet                   3.0.4                 py38_1003  
configparser              5.0.0                    pypi_0    pypi
cryptography              2.9.2            py38h1ba5d50_0  
curl                      7.71.1               he644dc0_4    conda-forge
cycler                    0.10.0                   py38_0  
cython                    0.29.21                  pypi_0    pypi
expat                     2.2.9                he6710b0_2  
fontconfig                2.13.1            h1056068_1002    conda-forge
freetype                  2.10.2               h5ab3b9f_0  
freexl                    1.0.5                h14c3975_0  
gdal                      3.1.2            py38hb61cb63_1    conda-forge
geographiclib             1.50                     pypi_0    pypi
geopy                     2.0.0                    pypi_0    pypi
geos                      3.8.1                he6710b0_0  
geotiff                   1.6.0                ha04d9d0_1    conda-forge
geoviews                  1.8.1                    pypi_0    pypi
gevent                    20.6.2                   pypi_0    pypi
giflib                    5.2.1                h516909a_2    conda-forge
glib                      2.65.0               h3eb4bd4_0  
greenlet                  0.4.16                   pypi_0    pypi
grequests                 0.6.0                    pypi_0    pypi
hdf4                      4.2.13               h3ca952b_2  
hdf5                      1.10.6          nompi_h3c11f04_101    conda-forge
holoviews                 1.13.3                   pypi_0    pypi
icu                       67.1                 he1b5a44_0    conda-forge
idna                      2.10                       py_0  
jinja2                    2.11.2                   pypi_0    pypi
jpeg                      9d                   h516909a_0    conda-forge
json-c                    0.13.1               h1bed415_0  
kealib                    1.4.13               h33137a7_1    conda-forge
kiwisolver                1.2.0            py38hfd86e86_0  
krb5                      1.17.1               h173b8e3_0  
lcms2                     2.11                 h396b838_0  
ld_impl_linux-64          2.33.1               h53a641e_7  
libblas                   3.8.0               17_openblas    conda-forge
libcblas                  3.8.0               17_openblas    conda-forge
libcurl                   7.71.1               hcdd3856_4    conda-forge
libdap4                   3.20.6               h1d1bd15_1    conda-forge
libedit                   3.1.20191231         h14c3975_1  
libev                     4.33                 h7b6447c_0  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.1.0                hdf63c60_0  
libgdal                   3.1.2                hb2a6f5f_1    conda-forge
libgfortran-ng            7.3.0                hdf63c60_0  
libiconv                  1.15                 h63c8f33_5  
libkml                    1.3.0             hb574062_1011    conda-forge
liblapack                 3.8.0               17_openblas    conda-forge
libnetcdf                 4.7.4           nompi_h84807e1_105    conda-forge
libnghttp2                1.41.0               hab1572f_1    conda-forge
libopenblas               0.3.10               h5a2b251_0  
libpng                    1.6.37               hbc83047_0  
libpq                     12.3                 h5513abc_0    conda-forge
libspatialite             4.3.0a            h57f1b35_1039    conda-forge
libssh2                   1.9.0                h1ba5d50_1  
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.1.0                h2733197_1  
libuuid                   2.32.1            h14c3975_1000    conda-forge
libwebp-base              1.1.0                h7b6447c_3  
libxcb                    1.14                 h7b6447c_0  
libxml2                   2.9.10               h72b56ed_2    conda-forge
lz4-c                     1.9.2                he6710b0_1  
markdown                  3.2.2                    pypi_0    pypi
markupsafe                1.1.1                    pypi_0    pypi
matplotlib-base           3.3.0            py38h91b0d89_1    conda-forge
mysql-connector-python    8.0.21                   pypi_0    pypi
ncurses                   6.2                  he6710b0_1  
numpy                     1.19.1           py38h30dfecb_0  
numpy-base                1.19.1           py38h75fe3a5_0  
olefile                   0.46                       py_0  
openjpeg                  2.3.1                h981e76c_3    conda-forge
openssl                   1.1.1g               h7b6447c_0  
owslib                    0.20.0                     py_0  
packaging                 20.4                     pypi_0    pypi
pandas                    1.1.0                    pypi_0    pypi
panel                     0.9.7                    pypi_0    pypi
param                     1.9.3                    pypi_0    pypi
pcre                      8.44                 he6710b0_0  
pillow                    7.2.0            py38hb39fc2d_0  
pip                       20.2.2                   py38_0  
pixman                    0.38.0            h516909a_1003    conda-forge
poppler                   0.89.0               h4190859_1    conda-forge
poppler-data              0.4.9                         0  
postgresql                12.3                 h8573dbc_0    conda-forge
proj                      7.1.0                h966b41f_1    conda-forge
protobuf                  3.12.4                   pypi_0    pypi
pthread-stubs             0.3                  h0ce48e5_1  
pycparser                 2.20                       py_2  
pyct                      0.4.6                    pypi_0    pypi
pyepsg                    0.4.0                      py_0  
pyopenssl                 19.1.0                     py_1  
pyparsing                 2.4.7                      py_0  
pyproj                    2.6.1.post1      py38h8e47818_1    conda-forge
pyshp                     2.1.0                      py_0  
pysocks                   1.7.1                    py38_0  
python                    3.8.5                hcff3b4d_0  
python-dateutil           2.8.1                      py_0  
python_abi                3.8                      1_cp38    conda-forge
pytz                      2020.1                     py_0  
pyviz-comms               0.7.6                    pypi_0    pypi
pyyaml                    5.3.1            py38h7b6447c_1  
readline                  8.0                  h7b6447c_0  
requests                  2.24.0                     py_0  
scipy                     1.5.0            py38habc2bb6_0  
setuptools                49.2.1                   py38_0  
shapely                   1.7.0            py38hd168ffb_3    conda-forge
six                       1.15.0                     py_0  
soupsieve                 2.0.1                    pypi_0    pypi
sqlite                    3.32.3               h62c20be_0  
tbb                       2020.0               hfd86e86_0  
tiledb                    2.0.8                h3effe38_0    conda-forge
tk                        8.6.10               hbc83047_0  
tornado                   6.0.4            py38h7b6447c_1  
tqdm                      4.48.2                   pypi_0    pypi
typing-extensions         3.7.4.2                  pypi_0    pypi
tzcode                    2020a                h516909a_0    conda-forge
urllib3                   1.25.10                    py_0  
wheel                     0.34.2                   py38_0  
xerces-c                  3.2.3                hfe33f54_1    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.10               h516909a_0    conda-forge
xorg-libsm                1.2.3             h84519dc_1000    conda-forge
xorg-libx11               1.6.11               h516909a_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.5                h7b6447c_0  
yaml                      0.2.5                h7b6447c_0  
zlib                      1.2.11               h7b6447c_3  
zope-event                4.4                      pypi_0    pypi
zope-interface            5.1.0                    pypi_0    pypi
zstd                      1.4.5                h9ceee32_0 
```

**Comment by jacobhutchinson** [comment]
Followed the advice from this issue regarding install problems here: https://github.com/holoviz/geoviews/issues/446. When I try ```conda install -c pyviz geoviews``` the environment still fails to solve as is the case with others in that thread, the advice of running ```conda install -c pyviz -c conda-forge geoviews``` worked. That being said, when I then try to upgrade from miniconda3 to full anaconda using ```conda install anaconda``` it states there is conflict between anaconda, prometheus_client, notebook, geoviews and holoeviews. Looks like it might be another install problem. Will try a fresh environment after it finishes examining the conflict.

**Comment by jacobhutchinson** [comment]
Closing this, as I realized it was in fact an installation error, probably in reference to the issues in https://github.com/holoviz/geoviews/issues/446. Re-installing conda and starting with a clean environment and re-installing using the command ```conda install -c pyviz -c conda-forge geoviews``` seemed to work. Something in my environment must have been stopping the render process somehow. I will still look into it a bit, but will close this.
